Python/math/euler_totient_function.py

```diff
@@ -0,0 +1,26 @@
+# python program for Euler's Totient Function 
+def euler_totient(q):
```

Comment:
Add docstring to make the code look well documented and add space before and after operators.